common/src/main/scala/explore/model/enums/SEDType.scala

```diff
@@ -73,7 +73,7 @@ sealed abstract class SEDTypeEnum[T](implicit
         _ =>
           EmissionLines[T](
             SortedMap.empty,
-            enumFDCUnits.all.head.withValueTagged(BigDecimal(1).refined[Positive])
+            enumFDCUnits.all.last.withValueTagged(BigDecimal(1).refined[Positive])
```

Comment:
I suggest we explicitly declare the default value.

Otherwise this may break if we reshuffle the enum order, or add a new one at the end.